test/autoupdate.test.js

```diff
@@ -2,94 +2,95 @@ require('./init.js');
 var assert = require('assert');
 var ds;
 
-before(function () {
+before(function() {
   ds = getDataSource();
 });
 
-describe('MS SQL server connector', function () {
-  it('should auto migrate/update tables', function (done) {
-
+describe('MS SQL server connector', function() {
+  it('should auto migrate/update tables', function(done) {
+    /* eslint-disable camelcase */
```

Comment:
I'm surprised we can't just search and replace the variable names. ie. change schema_v1 to schemaV1. Anyways, I trust your judgement, but its something we need to fix in the future.